src/algorithm/joint-configuration.hpp

```diff
@@ -17,6 +17,9 @@ namespace pinocchio
   /**
    * @brief      Integrate a configuration vector for the specified model for a tangent vector during one unit time
    *
+   * @details This function corresponds to the exponential map of the joint configuration Lie Group.
+   *          Its output can be interpreted as a sum in the Lie algebra \f$ q \oplus v \f$.
```

Comment:
I would say here: `as the sum from the Lie algebra to the joint configuration space`